contrib/babelfishpg_tsql/sql/upgrades/babelfishpg_tsql--2.2.0--2.3.0.sql

```diff
@@ -4229,3 +4229,128 @@ BEGIN
             RETURN NULL;
 END; $BODY$
 LANGUAGE plpgsql;
+
```

Comment:
Here, put several lines to deprecate the original views, for example
```
ALTER VIEW sys.server_principals RENAME TO server_principals_deprecated_in_2_3_0;
```
This is to avoid failures in MVU. When cx has some objects that are dependent on the original view, we are not able to directly drop them.